.github/workflows/test-kctrl-gh.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         ./hack/deploy-test.sh
 
         export KCTRL_E2E_NAMESPACE=kctrl-test
-        export KCTRL_E2E_IMAGE="localhost:5000/test-repo/testimage-$GITHUB_RUN_ID"
+        export KCTRL_E2E_IMAGE="`ifconfig | grep inet | grep -E '\b10\.' | awk '{ print $2}'`:5000/test-repo/testimage-$GITHUB_RUN_ID"
```

Comment:
What is the motivation behind doing this, won't the registry always run on localhost?

Reply:
So this we have done to eliminate the imgpkg circular registry error we are getting when push and pulling from the same registry and that goes away once we stop using localhost and start doing this.